client/components/FeedbacksCard/FeedbacksCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import FeedbackCard from './FeedbackCard';
+import './FeedbacksCard.scss';
+
+class FeedbacksCard extends Component {
+  render() {
+    const { feedbacksInfo } = this.props;
+
+    return (
+      <div className="feedbacks__wrap">
+        <p className="feedbacks__heading">
+          Feedbacks
+        </p>
+        <div className="feedbacks__body">
+          {feedbacksInfo.map((feedbackInfo, i) => (
+            <FeedbackCard feedbackInfo={feedbackInfo} key={i} id={i} />))}
```

Comment:
are you sure that  `i` is unique?

Reply:
I guess "i" is unique for all the items in the array. I can use https://www.npmjs.com/package/uniqid or some id from the database. Let me know what is the better solution.